### PR TITLE
[HackDay][Messanger]Improve PHP DocBlock for Messanger Stamps

### PR DESCRIPTION
**Pull request by Gramzivi**
| Q             | A
| ------------- | ---
| Branch?       | master 
| Bug fix?      | no
| New feature?  | yes/no 
| BC breaks?    | no     
| Deprecations? | no 
| Tests pass?   | yes    
| Fixed tickets |    
| License       | MIT
| Doc PR        | symfony/symfony-docs#... 

Add PHPDoc missing params for Messanger Stamps

### CODE REVIEW

**Comment by ogizanagi** [comment]
Thanks for taking the time to prepare and suggest these changes @Gramzivi. But actually we try to avoid adding "useless" phpdoc and you'll find a bunch of PR in this repository trying to remove any of this unnecessary docblocks (see #25859 for instance).

By "useless" here, I mean not-required like property types which are guessed by IDEs from constructor's typehints/docblock, redundant `@param` or `@return` types when there is already a typehint/return type, or comments not adding any value, like "SentStamp constructor" on the `SendStamp::__construct()` method.

I hope you'll understand and keep trying to contribute :)

File: src/Symfony/Component/Messenger/Stamp/HandledStamp.php [hunk 1]
```diff
@@ -23,8 +23,19 @@
  */
 final class HandledStamp implements StampInterface
 {
+    /**
+     * @var mixed
+     */
     private $result;
+
+    /**
+     * @var string
```
Comment (stof) [inline src/Symfony/Component/Messenger/Stamp/HandledStamp.php]: We don't add `@var` phpdoc on properties in Symfony, especially when the constructor allows IDEs to infer it already.

File: src/Symfony/Component/Messenger/Stamp/HandledStamp.php [hunk 2]
```diff
@@ -77,11 +94,17 @@ public function getResult()
         return $this->result;
     }
 
+    /**
+     * @return string
```
Comment (stof) [inline src/Symfony/Component/Messenger/Stamp/HandledStamp.php]: Please remove this phpdoc. We don't add it when it provides no value compared to the signature. Here, it only duplicates the return type.

Comment (stof) [reply src/Symfony/Component/Messenger/Stamp/HandledStamp.php]: actually, this applies to most places changed in that PR.

**Review by stof — CHANGES_REQUESTED** [review]

**Comment by nicolas-grekas** [comment]
Closing as explained, thanks for submitting.

**Pull request closed without merging**